deployability/modules/jobflow/tests/test_workflow_processor.py

```diff
@@ -8,17 +8,17 @@
 from unittest.mock import patch, MagicMock, call
 import pytest
 
-from workflow_engine.workflow_processor import WorkflowProcessor, DAG
-from workflow_engine.task import ProcessTask, TASKS_HANDLERS
+from jobflow.workflow_processor import WorkflowProcessor, DAG
```

Comment:
Shouldn't the name of the WorkflowProcessor class be changed?

Reply:
Modified in commit https://github.com/wazuh/wazuh-qa/pull/5460/commits/7910451c626e2a0b2a90e890e18ea93eb8b860a8